### PR TITLE
[LMENGA-1052] Update dependencies

### PR DESCRIPTION
**Pull request by Ashley-p44**
- Updated dependencies and removed deprecated packages
- Please verify by pulling down and running

### CODE REVIEW

File: node-react/package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "packages/*"
   ],
   "devDependencies": {
-    "lerna": "^3.20.2"
+    "lerna": "^4.0.0"
+  },
+  "engines": {
+    "node": ">=16.14.0"
```
Comment (ninablanson-p44) [inline node-react/package.json:16]: Could we bump this to be 17+? That's what we're using internally anyways.

**Review by ninablanson-p44 — COMMENTED** [review]
I think you also mentioned wanting to update the README to describe how to run this w/o Docker. Did you want to do that now or later?

**Comment by Ashley-p44** [comment]
@ninablanson-p44 updated the README

**Review by ninablanson-p44 — COMMENTED** [review]
I pulled this down and noticed a couple things:

**Using node 17**

For `yarn start`, we will need to add the flag `NODE_OPTIONS=--openssl-legacy-provider` to the `packages/frontend` command so that you don't get the envelopes error (if you're not seeing it, maybe it's something to do with M1 laptops...). There's a library that's acting up in there.

Since you bumped the node version to 17, you'll need to update the Docker base image used from `lts-slim` to `17-slim`, otherwise the build fails immediately due to the node versions not matching.

**Docker**

There's some issue with the `node_modules` not getting mounted that seems to be affecting the build. I can recreate it in the main branch, though, so it shouldn't block your PR - but we definitely need to look into this.

**Comment by ninablanson-p44** [comment]
@Ashley-p44 - I was able to resolve the Docker issue if you're willing to update the docker-compose file while you're in there. If you replace both instances of `/usr/src/app/node_modules` with `./node_modules:/usr/src/app/node_modules`, it works as expected.

**Comment by Ashley-p44** [comment]
@ninablanson-p44 thank you for helping with Docker, everything seems to be running now:
image

**Review by ninablanson-p44 — APPROVED** [review]

**Pull request merged**